terragrunt/aws/api/lambda.tf

```diff
@@ -18,6 +18,7 @@ module "api" {
   environment_variables = {
     SQLALCHEMY_DATABASE_URI = module.rds.proxy_connection_string_value
     FILE_QUEUE_BUCKET       = module.file-queue.s3_bucket_id
+    API_AUTH_TOKEN          = var.api_auth_token
```

Comment:
Could we change this to have the lambda passed the SSM parameter name `/scan-files/api_auth_token` and then have it look up the auth token value on cold starts?

Reply:
good call. more consistent with the way the rest of the envs are getting pulled in